cli/i18n/I18nFile.php

```diff
@@ -54,16 +54,24 @@ private function process(string $filename) {
 		$content = str_replace('<?php', '', $content);
 
 		$content = preg_replace([
-			"#',\s*//\s*TODO#i",
-			"#',\s*//\s*DIRTY#i",
-			"#',\s*//\s*IGNORE#i",
+			"#',\s*//\s*TODO.*#i",
+			"#',\s*//\s*DIRTY.*#i",
+			"#',\s*//\s*IGNORE.*#i",
```

Comment:
It's not necessarily a given that we want `.*`, though it seems likely. Alternatively:
```suggestion
			"#',\s*//\s*TODO$#i",
			"#',\s*//\s*DIRTY$#i",
			"#',\s*//\s*IGNORE$#i",
```

Reply:
Ups, that seems to fail

Reply:
That's good enough for me for now, although I would be interested in an explanation for why the `$` produces a bug here...

Reply:
Like I said I think .* should be fine, but I just wanted to emphasize I hadn't given it any thought besides the immediate situation.